scripts/restore-backup.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+set -e
+
+TMP="${TMP:-/ext/tmp}"
+# cannot use double quotes to allow bash to resolve * wildcards
+# shellcheck disable=SC2086
+filesArchive=$(find $TMP/*-archive-*.tar.gz)
+# shellcheck disable=SC2086
+databaseArchive=$(find $TMP/*-elife_2_0-mysql.gz)
+
+echo "Extracting $filesArchive"
+tar -xzf "$filesArchive" -C /ext/tmp
+
+echo "Restoring $filesArchive"
+rm -rf /srv/journal-cms/web/sites/default/files
```

Comment:
in order for this to work locally we should allow the web root to be overridden:

```
WEB="${WEB:-/srv/journal-cms/web}"
```

```
rm -rf $WEB/sites/default/files
mv $TMP/srv/journal-cms/web/sites/default/files $WEB/web/sites/default/
sudo chown -R www-data:www-data $WEB/sites/default/files
rm -rf $TMP/srv
```

Reply:
`ROOT` is now allowed